File: 0088-merge-sorted-array/0088-merge-sorted-array.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        ArrayList<Integer> ans = new ArrayList<>();
        for(int i=0; i<m; i++){
            ans.add(nums1[i]);
        }
        for(int v: nums2){
            ans.add(v);
        }
        
        Collections.sort(ans);
        int ptr = 0;
        for(int vi : ans){
            nums1[ptr] = vi;
            ptr++;
        }
        
    }
}

File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution(object):
    def merge(self, nums1, m, nums2, n):
        """
        :type nums1: List[int]
        :type m: int
        :type nums2: List[int]
        :type n: int
        :rtype: None Do not return anything, modify nums1 in-place instead.
        """
        ans = []
        for i in range(m):
            ans.append(nums1[i])
        for i in range(n):
            ans.append(nums2[i])
        for i in range(len(ans)):
            nums1[i] = ans[i]

        nums1.sort()
        
        